src/cli/et/commands/clean.py

```diff
@@ -0,0 +1,28 @@
+"""
+Clean CLI command removes generated files and directories.
+"""
+
+import os
+import shutil
+
+import click
+
+
+@click.command()
+@click.option("--all", is_flag=True, help="Remove the virtual environment as well.")
+def clean(all: bool):
+    """
+    Remove all generated files and directories from the repository.
+    ```
+    rm -rf .tox .cache .pytest_cache .mypy_cache fixtures build venv
+    ```
+    Note: The virtual environment is not removed by default.
+    """
+    # List of items to remove
+    items_to_remove = [".tox", ".pytest_cache", ".mypy_cache", "fixtures", "build", "site"]
+    if all:
+        items_to_remove.append("venv")
```

Comment:
Should this be [`.venv`](https://github.com/astral-sh/uv/blob/590294a0acc59c609aae82813b9ad9d22e7dbbbf/docs/pip/environments.md?plain=1#L12C85-L12C92)?

Reply:
Yup, this changed to `.venv` with the introduction of `uv` #777.

Reply:
I would consider moving `.tox` to `all` as well, btw. I very rarely delete `.tox`.

Reply:
I'm for moving `.tox` to all! Let's start it all fresh and clean 🧼